storage/src/vespa/storage/bucketdb/bucketmanager.h

```diff
@@ -34,16 +35,19 @@ class BucketManager : public StorageLinkQueued,
                       private framework::Runnable,
                       private framework::MetricUpdateHook
 {
+public:
     /** Type used for message queues */
-    typedef std::list<std::shared_ptr<api::StorageCommand> > CommandList;
-    typedef std::list<std::shared_ptr<api::RequestBucketInfoCommand> > BIList;
+    using CommandList = std::list<std::shared_ptr<api::StorageCommand> >;
+    using BucketInfoList = std::list<std::shared_ptr<api::RequestBucketInfoCommand> >;
```

Comment:
I think something like `BucketInfoRequestList` would be more intuitive here (or perhaps even better, `RequestsPerBucketSpace` since it does not encode the type in the name). Can also trim away space between `> >` at the end. Pre C++11 leftover.